### DISCUSSION TITLE
Kart uten ståscootere

### DISCUSSION THREAD

**Discussion by forteller** [entur/tavla#332] [Ideas]
Hadde vært fint om vi kunne hatt en egen boks med bare kart, og også om det var mulig å ha et kart i boksen med bysykkel, som viser hvor de stativene er.

**Comment by fridasolheim** [comment]
Hei @forteller, tusen takk for ønsket ditt! Vi er i gang med å implementere dette ☺️

**Comment by fridasolheim** [comment]
Hei igjen @forteller, 

Nå er kart implementert som en egen boks på Tavla. Takk for innspill! ☺️

**Comment by forteller** [comment]
Fantastisk! Veldig raskt levert. Takk!

**Comment by forteller** [comment]
Hei igjen. Jeg har oppdaget et par problemer med kartet.

Jeg liker å bruke tavla til å vise folk som skal komme og kjøpe noe brukt hos meg hvordan de lettest kommer seg til meg med kollektiv, i håp om at de skal droppe bilen. Men kartet gir et litt feil inntrykk som jeg frykter gjør det litt vanskeligere for folk, på disse to måtene:

- Det er kanskje bare hos meg, men den mørkeblå "tegnestiften" som viser adressen er på nabobygget, ikke på mitt bygg. Er det mulig å la meg dra og slippe den til et annet sted?
- Symbolet for buss og t-bane er samlet på ett sted, istedenfor å være der bussholdeplassene og t-banestasjonen faktisk er. Er det mulig å ha en funksjon for å velge å enten slå dem sammen slik som det er nå, eller plassere dem der hver enkelt ting faktisk er? Jeg forstår at noen steder er det mange bussholdeplasser fordelt over et litt større område, og at det fort kan bli rotete, men man kunne evt. hatt funksjon for å velge enten små buss-ikoner på hver holdeplass eller et større ikon midt mellom holdeplassene.

Takk!

**Reply by fridasolheim** [reply]
Hei igjen @forteller, takk for innspill! Vi noterer de og skal se på saken ☺️

**Comment by forteller** [comment]
Hei igjen. Jeg skulle bare sjekke om dette var fikset, og så at det ikke er det ennå. Det er greit (men jeg håper dere får gjort det en gang) men da oppdaget jeg også en annen ting: 

I standard størrelse viser ikke kartet opphavsrettsinformasjonen fra OSM. Det er bare en liten i nede i høyre hjørne, men om den er ment å rulle ut en info-boks ved mouseover eller noe, så skjer ikke det, for det er samme hjørne hvor man kan ta tak i boksen og endre størrelsen på den. Uansett er ikke en sånn løsning, hvor man må interagere med kartet for å se opphavsrettsinformasjonen, riktig måte å gjøre det på for et system som er laget for å ofte vises på skjermer folk ikke kan interagere med. 

Når man drar kartet ut i dobbel størrelse kommer boksen med informasjonen fram, men den viser ikke korrekt kreditering. Det skal så "© OpenStreetMap-bidragsyterene" eller "© OpenStreetMap contributors", ikke bare "© OpenStreetMap". Dette er et krav for å bruke kart fra OSM, så dette er viktig at gjøres riktig. Se https://www.openstreetmap.org/copyright

Takk! :)

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
